### PR TITLE
[IMP] Added reference and name_get for Business Requirement

### LINKED ISSUE #45: Add "Reference" in BR
Add "Reference" (Text data type) field to the BR and show on the Tree (list) view and adjust issue #37 to show this "Reference" with the description as _**[Reference] Description.**_ instead of [BR Number] Description

### PR DESCRIPTION
**Pull request by astirpe**
Fixes #45 and #37

### CODE REVIEW

File: business_requirement/tests/test_br.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ def test_get_level(self):
         br_vals1 = {
             'name': ' test',
             'description': 'test',
+            'ref': 'Ref1',
```
Comment (dreispt) [inline business_requirement/tests/test_br.py]: Usual we use `/` for the default value in reference fields.

**Review by dreispt — COMMENTED** [review]
LGTM :+1:

**Comment by astirpe** [comment]
@elicoidal May I ask you to take a look at this PR?

File: business_requirement/tests/test_br.py (hunk 1 above)
Comment (elicoidal) [reply business_requirement/tests/test_br.py]: Agree

**Comment by elicoidal** [comment]
slight change on the void reference case. Other than that :+1:

**Comment by astirpe** [comment]
Rebased branch

**Comment by elicoidal** [comment]
Good to go.
Thanks!

**Comment by astirpe** [comment]
Thank you!
We need to remember to port these changes to V9 also.

**Comment by elicoidal** [comment]
Yes indeed... 
How should we control that? create a WIP PR for the moment?

**Comment by astirpe** [comment]
PR for porting the changes to V9, here it is: #68

**Pull request merged**